.travis.yml

```diff
@@ -3,5 +3,9 @@ language: python
 python:
   - "3.6"
 cache: pip
-install: "make"
-script: make ci
+install:
+    - "make"
+    - pip install coveralls
```

Comment:
Ich frage mich, ob das in der Makefile sein sollte. Aber es ist auch Travis spezifisch. Hm.